bitnami/redis/values.yaml

```diff
@@ -1371,6 +1371,9 @@ metrics:
     ## @param metrics.service.port Redis&trade; exporter service port
     ##
     port: 9121
+    ## @param metrics.service.name Redis&trade; exporter service port name
+    ##
+    name: http-metrics
```

Comment:
To be in sync with our catalog, could you rename this param to `portName`?

Reply:
yes